spec/integration/http_integration_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+require_relative 'rpc_contract'
+
+module RPC
+  class Client < Protein::Client
+    transport :http,
+      url: ENV['RPC_SERVER_URL'],
+      secret: ENV['RPC_SECRET']
+
+    route "RPC::Router"
+  end
+end
+
+RSpec.describe "HTTP transport integration" do
```

Comment:
Considering the number of test cases may grow - wouldn't it be better to somehow reuse the test cases across multiple transport mechanisms instead of duplicating same tests for every transport spec?

Reply:
I switched to shared examples